dp/drivers/ixgbe.c

```diff
@@ -1113,8 +1114,12 @@ int ixgbe_init(struct pci_dev *pci_dev, struct ix_rte_eth_dev **ethp)
 
 	if (!strcmp(driver->name, "rte_ixgbe_pmd"))
 		dev->dev_ops = &eth_dev_ops;
-	else if (!strcmp(driver->name, "rte_ixgbevf_pmd"))
+	else if (!strcmp(driver->name, "rte_ixgbevf_pmd")){
 		dev->dev_ops = &vf_eth_dev_ops;
+		/* check that the config is right */
+		if (CFG.num_cpus > 4)
+			log_warn("using more than 4 cores on a VF is currently not supported.\n");
```

Comment:
* If we don't support, then why not panic instead of log_warn? Will IX run correctly with 5 cores? What can the user expect if running with 5 cores.
* Maybe we can't support 4 cores because we don't have enough 2MB pages. Have you tried increasing this number?
* Is the number 4 dependent on the number of running IX on VFs? Have you tested with 3 IX each with 4 cores?